lib/presentation/extensions/go_router_extensions.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:go_router/go_router.dart';
+
+extension GoRouterExtensions on GoRouter {
+  String? get activeRoomId {
+    final path = routeInformationProvider.value.uri.path;
+    if (!path.startsWith('/rooms/')) return null;
+    return path.split('/')[2];
```

Comment:
check the length of split function first, cause it may throw at runtime

Reply:
Fixed